build.gradle

```diff
@@ -2,19 +2,18 @@ buildscript {
     repositories {
         mavenCentral()
         mavenLocal()
-        jcenter()
```

Comment:
why are we removing this repo?

Reply:
This jcenter is out of date https://blog.gradle.org/jcenter-shutdown#:~:text=Based%20on%20the%20current%20timeline,you%20continue%20to%20use%20JCenter.

Reply:
BTW, in this PR, I am upgrading gradle 8 as well.